ci/310-latest.yaml

```diff
@@ -3,59 +3,50 @@ channels:
   - conda-forge
 dependencies:
   - python=3.10
-  - six
-  - pyyaml
-  - deprecated
-  # required
-  # testing
-  - matplotlib
-  - pytest
-  - pytest-cov
-  - coverage
-  - codecov
-  # optional
+  # ecosystem
+  - access
+  - esda
+  - giddy
+  - inequality
+  - libpysal
+  - mapclassify
+  - mgwr
+  - momepy
+  - pointpats
+  - segregation
+  - spaghetti
+  - splot
+  - spint
+  - spopt
+  - spreg
+  - spvcm
+  - tobler>=0.2.1
+  # external & optiopnal
   - bokeh
-  - scikit-learn
+  - deprecated
+  - dill
+  - fiona
```

Comment:
Do we need Fiona at all?

Reply:
I mean, in the latest. We need that pin in the oldest.

Reply:
suppose not?